app/src/js/components/Rules/add.js

```diff
@@ -45,11 +30,48 @@ ModalBody.propTypes = {
   record: PropTypes.object
 };
 
-const AddRule = ({ rules, ...rest }) => {
+const AddRule = ({ rules, location = {}, dispatch, schema, ...rest }) => {
+  const [defaultValue, setDefaultValue] = useState({
+    name: '',
+    workflow: '',
+    provider: '',
+    collection: {
+      name: '',
+      version: ''
+    },
+    meta: {},
+    rule: {
+      type: '',
+      value: ''
+    },
+    state: 'ENABLED'
+  });
```

Comment:
Might be a little easier to read if we kept the filler object outside of the component with a different variable name and just set it here like `const [defaultValue, setDefaultValue] = useState(defaultRuleObject);`, but doesn't really matter

Reply:
sure